src/components/Space/GenericSpace.tsx

```diff
@@ -12,46 +12,55 @@ interface GenericSpace {
   inClosedSet: boolean;
 }
 
-const GenericSpace: React.FC<GenericSpace> = (props) => {
-  if (
-    props.inOpenSet &&
-    !props.path &&
-    props.type !== SpaceTypes.start &&
-    props.type !== SpaceTypes.end
-  ) {
-    return (
-      <mesh
-        position={new Vector3(0, -0.5, 0)}
-        rotation={[-(Math.PI / 2), 0, 0]}
-      >
-        <planeBufferGeometry attach="geometry" args={[1, 1, 1, 1]} />
-        <meshPhongMaterial attach="material" color="green" />
-      </mesh>
-    );
-  } else if (
-    props.inClosedSet &&
-    !props.path &&
-    props.type !== SpaceTypes.start &&
-    props.type !== SpaceTypes.end
-  ) {
-    return (
-      <mesh
-        position={new Vector3(0, -0.5, 0)}
-        rotation={[-(Math.PI / 2), 0, 0]}
-      >
-        <planeBufferGeometry attach="geometry" args={[1, 1, 1, 1]} />
-        <meshPhongMaterial attach="material" color="orange" />
-      </mesh>
-    );
-  } else if (props.type === "empty") {
-    return <Empty visited={props.visited} path={props.path} />;
-  } else {
-    return (
-      <Suspense fallback="none">
-        <Wall type={props.type} />
-      </Suspense>
-    );
-  }
+// <<<<<<< Updated upstream
+// const GenericSpace: React.FC<GenericSpace> = (props) => {
+//   if (
+//     props.inOpenSet &&
+//     !props.path &&
+//     props.type !== SpaceTypes.start &&
+//     props.type !== SpaceTypes.end
+//   ) {
+//     return (
+//       <mesh
+//         position={new Vector3(0, -0.5, 0)}
+//         rotation={[-(Math.PI / 2), 0, 0]}
+//       >
+//         <planeBufferGeometry attach="geometry" args={[1, 1, 1, 1]} />
+//         <meshPhongMaterial attach="material" color="green" />
+//       </mesh>
+//     );
+//   } else if (
+//     props.inClosedSet &&
+//     !props.path &&
+//     props.type !== SpaceTypes.start &&
+//     props.type !== SpaceTypes.end
+//   ) {
+//     return (
+//       <mesh
+//         position={new Vector3(0, -0.5, 0)}
+//         rotation={[-(Math.PI / 2), 0, 0]}
+//       >
+//         <planeBufferGeometry attach="geometry" args={[1, 1, 1, 1]} />
+//         <meshPhongMaterial attach="material" color="orange" />
+//       </mesh>
+//     );
+//   } else if (props.type === "empty") {
+//     return <Empty visited={props.visited} path={props.path} />;
+//   } else {
+//     return (
+//       <Suspense fallback="none">
+//         <Wall type={props.type} />
+//       </Suspense>
+//     );
+//   }
+// =======
+const GenericSpace: React.FC<GenericSpace> = props => {
+  return(
+    <Suspense fallback="none">
+      {props.type === 'empty' ? <Empty visited={props.visited} path={props.path} /> : <Wall type={props.type} />}
+    </Suspense>
+  );
+// >>>>>>> Stashed changes
```

Comment:
This looks like a merge conflict, is this intended?

Reply:
Yes, sorry, this is mike's code for coloring closed sets and open sets. I should have made an inline comment about it - It's intentionally left in for future reference if and when we texture open and closed sets.

Reply:
Ah ok cool

Reply:
ye, lookin good :)